src/include/distributed/commands.h

```diff
@@ -56,6 +56,7 @@ typedef struct DistributeObjectOps
 	List * (*preprocess)(Node *, const char *, ProcessUtilityContext);
 	List * (*postprocess)(Node *, const char *);
 	ObjectAddress (*address)(Node *, bool);
+	bool markDistributed;
```

Comment:
what is our criteria to set `markDistributed = true`? Would be nice to have some comment

Reply:
Added